packages/react/src/hooks/useAuth.ts

```diff
@@ -21,6 +21,9 @@ import { createGetToken, createSignOut } from './utils';
  *
  * The following example demonstrates how to use the `useAuth()` hook to access the current auth state, like whether the user is signed in or not. It also includes a basic example for using the `getToken()` method to retrieve a session token for fetching data from an external resource.
  *
+ * <Tabs items='React,Next.js'>
```

Comment:
If you use these MDX components (they are not resolved, just passed along) in such way, they won't be rendered in IntelliSense